[DOC-5508][DOC-7977]  Improve PowerShell commands for Windows quickstart and install

mdlinville opened the pull request:
[DOC-5508][DOC-7977] Improve PowerShell commands for Windows quickstart and install

## Previews
[src/current/cockroachcloud/deploy-a-python-to-do-app-with-flask-kubernetes-and-cockroachcloud.md](https://deploy-preview-17355--cockroachdb-docs.netlify.app/docs/cockroachcloud/deploy-a-python-to-do-app-with-flask-kubernetes-and-cockroachcloud.html)
[src/current/cockroachcloud/quickstart-trial-cluster.md](https://deploy-preview-17355--cockroachdb-docs.netlify.app/docs/cockroachcloud/quickstart-trial-cluster.html)
[src/current/cockroachcloud/connect-to-your-cluster.md](https://deploy-preview-17355--cockroachdb-docs.netlify.app/docs/cockroachcloud/connect-to-your-cluster.html)
[src/current/v22.2/install-cockroachdb-windows.md](https://deploy-preview-17355--cockroachdb-docs.netlify.app/docs/v22.2/install-cockroachdb-windows.html)
[src/current/v23.1/install-cockroachdb-windows.md](https://deploy-preview-17355--cockroachdb-docs.netlify.app/docs/v23.1/install-cockroachdb-windows.html)

## Tests
- [x] Local build
- [x] Linkcheck

gemma-shay reviewed (commented):
Sorry for delayed review! These changes look good to me, but I don't have access to a windows virtual machine anymore to test the powershell commands. Did you or someone else do testing?

ianjevans commented:
> Sorry for delayed review! These changes look good to me, but I don't have access to a windows virtual machine anymore to test the powershell commands. Did you or someone else do testing?

FWIW, my AWS Workspaces Windows login still works. I think you (Gemma) had an account created at the same time. My email for the login was sent 10/4/21.

Anyway, if you need me to test on Windows I can do so.

ianjevans commented on src/current/_includes/cockroachcloud/download-the-binary.md:
```diff
@@ -18,11 +20,13 @@
 
     <section class="filter-content" markdown="1" data-scope="windows">
 
+    If you have not done so already, use PowerPhell to run the first command in the dialog, which is a Powershell script that installs the CockroachDB binary and adds its location in the system `PATH`:
```
```suggestion
    If you have not done so already, use PowerShell to run the first command in the dialog, which is a PowerShell script that installs the CockroachDB binary and adds its location in the system `PATH`:
```

ianjevans reviewed (approved):
Some typos, but otherwise it looks good. LMK if you need me to verify the commands on a Windows VM.

mdlinville commented:
@ianjevans It would certainly save me needing to set all that up. You can invoke a script like `powershell.exe -Command <path_to_file>`. This one is tricky to turn into a one-line because it has a mix of single and double quotes in the commands, and the whole command needs to be quoted! We don't do much testing of Windows runtimes, so any testing we do of the command is probably above and beyond.

ianjevans commented:
Looks like there's issues with the rendering of the code blocks.
<img width="1089" alt="Screenshot 2023-07-06 at 9 51 40 AM" src="https://github.com/cockroachdb/docs/assets/5760366/29388613-d8f3-42d2-971e-30049f8eac1a">

mdlinville commented:
> Looks like there's issues with the rendering of the code blocks. <img alt="Screenshot 2023-07-06 at 9 51 40 AM" width="1089" src="https://user-images.githubusercontent.com/5760366/251518298-29388613-d8f3-42d2-971e-30049f8eac1a.png">

The `<section>` tags were missing `markdown="1"`. Fixed. In the meantime, I converted more of the HTML to Markdown while troubleshooting.

ianjevans reviewed (approved):
Forgot to finish this, but I tested the scripts and they all work.
